### ISSUE TITLE
Found unclaimed freebie(s)

### ISSUE THREAD

**Issue by MotiasF33** [Revadike/epicgames-freebies-claimer#187]
### Expected Behavior
No unclaimed freebie(s) found 

### Actual Behavior
When I try to claim the game that I already have for this account, it ever says: "Found n unclaimed freebie(s)" and error
![image](https://user-images.githubusercontent.com/47714084/148696341-414368b8-ea2c-41c3-b928-45a644bcc76f.png)
```
 2022-01-09 | 18:37:09.411 | INFO  | Epicgames Freebies Claimer (1.5.8) by Revadike
 2022-01-09 | 18:37:10.606 | INFO  | Found 1 unclaimed freebie(s) for *
 2022-01-09 | 18:37:11.746 | INFO  | Logged in as Motiasf33 (*)
 2022-01-09 | 18:37:13.111 | ERROR | Failed to claim Gods Will Fall (Error: We could not confirm your purchase at this time.  Please try again later.)
 2022-01-09 | 18:37:13.168 | INFO  | Logged Motiasf33 out of Epic Games
```
### Specifications

  - `epicgames-freebies-claimer` Version: 1.5.8
  - Platform / OS: Win 10
  - Node Version: 15.7.0

**Comment by adi170-alt** [comment]
Dupe of #172.

**Comment by Revadike** [comment]
It's not.

**Comment by adi170-alt** [comment]
> It's not.

owh sorry, I didn't read the full log

**Comment by jackblk** [comment]
@Revadike instead of using `history.json` we can check in the account via client, func `findActiveEntitlementByName` or get all with `getEntitlements`

**Comment by Revadike** [comment]
True, but it will require an extra request, which is why I avoided it until now, but I agree this is what we should do.

**Comment by MotiasF33** [comment]
If game unclaimed, I have same error...
`
> epicgames-freebies-claimer@1.5.8 start
> node claimer.js

 2022-01-13 | 18:01:38.979 | INFO  | Epicgames Freebies Claimer (1.5.8) by Revadike
 2022-01-13 | 18:01:41.247 | INFO  | Found 1 unclaimed freebie(s) for email@yandex.ru
 2022-01-13 | 18:01:42.246 | INFO  | Logged in as Motiasf33 (uid)
 2022-01-13 | 18:01:43.768 | ERROR | Failed to claim Galactic Civilizations III (Error: We could not confirm your purchase at this time.  Please try again later.)
 2022-01-13 | 18:01:43.845 | INFO  | Logged Motiasf33 out of Epic Games`

![image](https://user-images.githubusercontent.com/47714084/149384770-860d84b6-fb3a-4545-b956-0bf04fbba525.png)
I don't know, claimer just doesn't work for me

**Comment by baduhai** [comment]
Just got the same error as @MotiasF33.

**Comment by vmhocuron0** [comment]
got the same error as @MotiasF33. the same game actually ¯\_(ツ)_/¯
please help

**Comment by aleksstevcic** [comment]
Can confirm, also get the same error.

**Comment by gabriel-lando** [comment]
Same here as @MotiasF33 

```
 2022-01-24 | 16:35:09.726 | INFO  | Epicgames Freebies Claimer (1.5.8) by Revadike
 2022-01-24 | 16:35:14.400 | INFO  | Found 1 unclaimed freebie(s) for email@email.com
 2022-01-24 | 16:35:18.856 | INFO  | Logged in as USER (UUID)
 2022-01-24 | 16:35:34.654 | ERROR | Failed to claim Relicta (Error: We could not confirm your purchase at this time.  Please try again later.)
 2022-01-24 | 16:35:35.359 | INFO  | Logged USER out of Epic Games
```

![image](https://user-images.githubusercontent.com/18423120/150825296-36733d3f-ddea-4e1e-825f-35fb11f1e019.png)

**Comment by NorskNoobing** [comment]
Still no fix?
```2022-01-27 | 20:01:11.873 | INFO  | Epicgames Freebies Claimer (1.5.8) by Revadike
 2022-01-27 | 20:01:13.801 | INFO  | Found 1 unclaimed freebie(s) for $mail
 2022-01-27 | 20:01:17.476 | INFO  | Logged in as NorskNoobing ($uid)
 2022-01-27 | 20:01:20.681 | ERROR | Failed to claim DAEMON X MACHINA (Error: We could not confirm your purchase at this time.  Please try again later.)
 2022-01-27 | 20:01:21.273 | INFO  | Logged NorskNoobing out of Epic Games
 2022-01-27 | 20:01:22.075 | INFO  | Push notification sent

**Comment by Revadike** [comment]
Sorry

**Comment by LudovicOmarini** [comment]
Same here

` 2022-02-02 | 03:46:39.271 | INFO  | Epicgames Freebies Claimer (1.5.8) by Revadike
 2022-02-02 | 03:46:42.743 | INFO  | Found 1 unclaimed freebie(s) for omarini.ludovic@gmail.com
 2022-02-02 | 03:46:46.300 | INFO  | Logged in as Lunoh (527433b2081844e0aa13bb75f28a25fc)
 2022-02-02 | 03:46:50.346 | ERROR | Failed to claim DAEMON X MACHINA (Error: We could not confirm your purchase at this time.  Please try again later.)
 2022-02-02 | 03:46:50.911 | INFO  | Logged Lunoh out of Epic Games`

**Comment by LudovicOmarini** [comment]
Same here

>  2022-02-02 | 03:46:39.271 | INFO  | Epicgames Freebies Claimer (1.5.8) by Revadike
 2022-02-02 | 03:46:42.743 | INFO  | Found 1 unclaimed freebie(s) for omarini.ludovic@gmail.com
 2022-02-02 | 03:46:46.300 | INFO  | Logged in as Lunoh (527433b2081844e0aa13bb75f28a25fc)
 2022-02-02 | 03:46:50.346 | ERROR | Failed to claim DAEMON X MACHINA (Error: We could not confirm your purchase at this time.  Please try again later.)
 2022-02-02 | 03:46:50.911 | INFO  | Logged Lunoh out of Epic Games

**Comment by LudovicOmarini** [comment]
Same here

>  2022-02-02 | 03:46:39.271 | INFO  | Epicgames Freebies Claimer (1.5.8) by Revadike
 2022-02-02 | 03:46:42.743 | INFO  | Found 1 unclaimed freebie(s) for $mail
 2022-02-02 | 03:46:46.300 | INFO  | Logged in as $login ($uid)
 2022-02-02 | 03:46:50.346 | ERROR | Failed to claim DAEMON X MACHINA (Error: We could not confirm your purchase at this time.  Please try again later.)
 2022-02-02 | 03:46:50.911 | INFO  | Logged $login out of Epic Games

**Comment by cryptedbv** [comment]
Same issue here

```
epic_free    |  2022-02-02 | 09:22:35.572 | INFO  | Epicgames Freebies Claimer (1.5.8) by Revadike
epic_free    |  2022-02-02 | 09:22:38.065 | INFO  | Found 1 unclaimed freebie(s) for *************@**********.com
epic_free    |  2022-02-02 | 09:22:40.614 | INFO  | Logged in as ********** (*******************************)
epic_free    |  2022-02-02 | 09:22:43.535 | ERROR | Failed to claim DAEMON X MACHINA (Error: We could not confirm your purchase at this time.  Please try again later.)
epic_free    |  2022-02-02 | 09:22:43.941 | INFO  | Logged ******** out of Epic Games
epic_free    |  2022-02-02 | 09:22:43.943 | INFO  | Waiting 1440 minutes
```

**Comment by Maximapple** [comment]
Just got the same error. Will it be fixed soon?

**Comment by L1cardo** [comment]
I am having the same problem here.
<img width="1314" alt="image" src="https://user-images.githubusercontent.com/33802186/153143557-3eeb10dd-0e51-4543-b16e-2adab879089a.png">

**Comment by NorskNoobing** [comment]
> Around the time this script stopped working they started showing me a quite challenging captcha every time I claim the free game using the web interface. If they now show the captcha everytime, I doubt that this script can be fixed soon.

I only need to press place order, no captcha or anything

**Comment by QIN2DIM** [comment]
I nailed it gracefully.

**Comment by jackblk** [comment]
> I only need to press place order, no captcha or anything

The captcha is implemented in the API call. It does the calculation to see if you're a robot and add signatures to the API call.

You don't see any captcha because your trust score is high, it still has signature in the API call though. If your trust score is low, it will show the captcha.

We cannot calculate the signature because it's hard. You can use the browser for that but it's not direct API call anymore, it will be just like any other project, using a browser to claim.

**Comment by Revadike** [comment]
Please keep off-topic stuff out of this issue.

**Comment by ProBlacKirito** [comment]
from the browser it asks me nothing but get and buy, but I find the problem that it does not make me take it (sorry for my englis
![Immagine 2022-03-19 160205](https://user-images.githubusercontent.com/74011684/159126684-95b83dad-dec7-4951-8b74-9e65d4d0fbf8.png)
h
